astropy/utils/compat/numpycompat.py

```diff
@@ -4,12 +4,22 @@
 earlier versions of Numpy.
 """
 
+from types import ModuleType
```

Comment:
I don't think the changes in `numpycompat` are used in this PR, so it should be separate.